addon/templates/components/cfb-form-editor/question.hbs

```diff
@@ -121,6 +121,18 @@
           required=true
           renderComponent=(component "cfb-form-editor/question/options")
         }}
+
+        {{f.input
+          name="widgetType"
+          label=(t "caluma.form-builder.question.widgetType")
+          required=true
+          class="uk-flex uk-flex-between uk-flex-column"
+
+          type="select"
+          optionTargetPath="value"
+          optionLabelPath="label"
+          options=(array (hash value="default" label="Default") (hash value="powerselect" label="Dropdown"))
```

Comment:
If you use `radio` instead of `default` you can drop the conditional in cf-field/input.js:
`(customtype && customtype !== "default" && customtype)`

Reply:
I think calling it "Radio" instead of "Default" also makes more sense for the user, especially if the question type is "Choice" and not "Radio".

Reply:
The problem with a specific default type is that the widget is also used for the MultipleChoiceQuestion type.

Reply:
You could make the options dynamic though, right?

Reply:
In theory definitely. But I'm not sure it will make the code shorter/better/easier. Although having the hash defined in the component instead of the template would be cleaner. I'm just not sure about the dynamic content.